_includes/cusdis.html

```diff
@@ -0,0 +1,8 @@
+<div class="comments" id="cusdis"
+  data-host="https://cusdis.com"
+  data-app-id={{site.cusdis_id}}
```

Comment:
Misssing double quote?

Reply:
Yep. Among other stuff. Tested and fixed it in my last commit.
It also works with localhost. May be a good idea to restrict domain in the future to avoid spam/dos.